bootstrap/src/main/java/io/airlift/bootstrap/Bootstrap.java

```diff
@@ -178,7 +210,7 @@ public Set<String> configure()
 
         // replace environment variables in property values
         List<Message> errors = new ArrayList<>();
-        properties = replaceEnvironmentVariables(properties, System.getenv(), (key, error) -> {
+        configurationResolver.getResolvedConfiguration(properties, (key, error) -> {
```

Comment:
@electrum Do we need to migrate the environment config resolution to this framework ?

Reply:
That seems like a good idea. It will probably simplify the code and is a good test for the framework.